### PR TITLE
Upgrade Quarkus to 1.8.0.Final

### PR DESCRIPTION
**Pull request by rsynek**
**referenced Pull Requests**:

https://github.com/kiegroup/optaplanner/pull/922

### CODE REVIEW

File: quarkus-school-timetabling/build.gradle [hunk 1]
```diff
@@ -65,3 +65,12 @@ java {
 test {
     systemProperty "java.util.logging.manager", "org.jboss.logmanager.LogManager"
 }
+
+repositories {
+    maven {
+        url "https://repository.jboss.org/nexus/content/groups/public/"
+        mavenContent {
+            snapshotsOnly()
```
Comment (ge0ffrey) [inline quarkus-school-timetabling/build.gradle:73]: Why does the gradle build needs this and the maven builds don't?

Ideally when we do a release, the "stable" branch doesn't have unnecessary code like this, as we expect many users to start from these quickstarts.

Comment (rsynek) [reply quarkus-school-timetabling/build.gradle:73]: Maven builds do need it too, see https://github.com/kiegroup/optaplanner-quickstarts/blob/development/quarkus-school-timetabling/pom.xml#L182.

The reason is that now we depend on kogito snapshots.

Comment (ge0ffrey) [reply quarkus-school-timetabling/build.gradle:73]: Argh, we'll need to delete this part everytime we sync the development branch to the stable branch (= default branch).

Comment (rsynek) [reply quarkus-school-timetabling/build.gradle:73]: That's one more step for the release pipeline, I guess.

**Pull request merged**